guardian/commercial-templates
# Bump eslint, prettier, typescript and related plugins

## What does this change?

Bumps:

- ESLint
- Prettier
- TypeScript

## How to test

Ensure the following commands continue to execute successfully:

```sh
$ npm run check
$ npm run format
$ npm run lint
```
